tests/run/cpp_stl_atomic.pyx

```diff
@@ -0,0 +1,106 @@
+# distutils: extra_compile_args=-std=c++11
+# mode: run
+# tag: cpp, werror
```

Comment:
Instead of the distutils directive above, there should simply be a tag `cpp11` here, which does the same in the test runner, but additionally excludes the test if C++11 is not available.